machine-learning/app/models/constants.py

```diff
@@ -66,6 +66,65 @@
 SUPPORTED_PROVIDERS = ["CUDAExecutionProvider", "OpenVINOExecutionProvider", "CPUExecutionProvider"]
 
 
+WEBLATE_TO_FLORES200 = {
```

Comment:
I wonder where this mapping should live. To some extent the server knows which model it is using when it sends the request so you could argue this belongs there. What do you think?

Reply:
I'd say it's better for it to live here - it makes things easier if you query ML directly or use the models in another context like in notebooks.